exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/QaSchemaIntegrationTest.java

```diff
@@ -39,20 +35,19 @@ static void loadLibrary() {
   }
 
   @Test
-  void getStateHashesEmptyDb() throws CloseFailuresException {
-    try (Database db = MemoryDb.newInstance();
-         Cleaner cleaner = new Cleaner()) {
-      Snapshot snapshot = db.createSnapshot(cleaner);
-      QaSchema schema = new QaSchema(snapshot);
+  void getStateHashesEmptyDb() {
+    try (TestKit testKit = TestKit.forService(QaServiceModule.class)) {
```

Comment:
Do we have a task for implementing testkit extension for junit? 🙃

Reply:
Haven't you been to the grooming?

Reply:
> Do we have a task for implementing testkit extension for junit? 🙃

Yes, we do :) ECR-3076 I believe